src/old-components/Guilds/ActionsBuilder/CallDetails/index.tsx

```diff
@@ -54,6 +54,10 @@ const CallDetails: React.FC<ActionViewProps> = ({
     }
 
     if (type.startsWith('uint') || type.startsWith('int')) {
+      if (Array.isArray(value)) {
+        return <ParamDetail>{BigNumber.from(value[0]).toString()}</ParamDetail>;

```

Comment:
We could probably just add this to a ternary isArray ? value[0] : value

Reply:
Actually, we should instead make a loop. When in future we support arrays for permissions we will want to show all of them.
If it is an array you can loop over converting each to a string and keep the square brackets [123, 456]